ui/installer/VCSA/configs

```diff
@@ -10,3 +10,6 @@ VIC_UI_HOST_URL="NOURL"
 
 # If VIC_UI_HOST_URL starts with "https", enter SHA-1 thumbprint of the web server hosting the vic-ui.zip file
 VIC_UI_HOST_THUMBPRINT=""
```

Comment:
@andrewtchin great point. i think the space character is the expected delimiter, but i'll double check. also i'll give it try with `:` to see if VC doesn't like it. thanks for your input